### PR TITLE
Export causatives

### PR DESCRIPTION
**Pull request by moonso**

### CODE REVIEW

File: scout/commands/base.py [hunk 1]
```diff
@@ -48,7 +49,8 @@
 @click.pass_context
 def cli(context, mongodb, username, password, authdb, host, port, loglevel, config, demo):
     """scout: manage interactions with a scout instance."""
-    coloredlogs.install(level=loglevel)
+    log_format = "%(message)s" if sys.stdout.isatty() else None
+    coloredlogs.install(level=loglevel, fmt=log_format)
```
Comment (robinandeer) [inline scout/commands/base.py]: Nice - would be better with colored text for "info" logs to make it stand out but it's work in progress 🙂

File: scout/export/variant.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
+from pprint import pprint as pp
 
 
 def export_causatives(adapter, collaborator):
```
Comment (robinandeer) [inline scout/export/variant.py]: Should we perhaps add the option to pass a list of collaborators to export causative variants from multiple institutes? Could do in separate PR

File: scout/export/variant.py [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+        collaborator(str)
+    """
+    #put variants in a dict to get unique ones
     variant_string = ("{0}\t{1}\t{2}\t{3}\t{4}\t{5}\t{6}\t{7}")
     
-    for variant_id in variants:
-        variant = variants[variant_id]
+    for document_id in adapter.get_causatives(institute_id=collaborator):
+        variant_obj = adapter.variant(document_id)
         yield variant_string.format(
```
Comment (robinandeer) [inline scout/export/variant.py]: are all these fields garuanteed to exist?

File: scout/export/variant.py (hunk 2 above)
Comment (moonso) [reply scout/export/variant.py]: Yes I also thought about that. One should also be able to export all variants from all institutes.

File: scout/export/variant.py (hunk 3 above)
Comment (moonso) [reply scout/export/variant.py]: Jepp

**Comment by moonso** [comment]
Just realized that the variants are not exported in a sorted way...

**Comment by moonso** [comment]
Guess I have to fix that

**Comment by robinandeer** [comment]
Does it matter that they are sorted? Sorted on RankScore or chromosome:pos?

**Comment by robinandeer** [comment]
It's fine to merge!

**Comment by moonso** [comment]
They should allways be sorted on chromosome:pos
